docs/roman/pipeline/calroman_level2.rst

```diff
@@ -0,0 +1,81 @@
+.. _calroman_level2:
+
+
+calroman_level2: Stage 2 Detector Processing
+==============================================
+
+:Class: `romancal.pipeline.Level2Pipeline`
+:Alias: calroman_level2
+
+The ``Level2Pipeline`` applies basic detector-level corrections to given exposure
+types (imaging, prism, and grism.). It is applied to one
+exposure at a time.
+It is sometimes referred to as "ramps-to-slopes" processing, because the input raw data
+are in the form of one or more ramps (integrations) containing accumulating
+counts from the non-destructive detector readouts and the output is a corrected
+countrate (slope) image.
```

Comment:
This sentence needs to be reworded as for Roman `ramps-to-slopes` is only part of L2.